frontend/src/components/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import styles from "./Button.module.css";
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export interface ButtonProps extends React.ComponentProps<"button"> {
+  label: string;
+  kind?: "primary" | "secondary";
+}
+
+export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(function Button(
```

Comment:
Andrew created a Button component that's merged to main already. Let's use that instead.

Reply:
Using Andrew's button instead!